### DISCUSSION TITLE
Dark mode in @apply

### DISCUSSION THREAD

**Discussion by alikenski** [tailwindlabs/tailwindcss#3609] [Help]
Can We use dark mode **dark:** in _apply_?

**Comment by simonswiss** [comment]
Hey!

Yep you can - here's an example of that in this Tailwind Play:

https://play.tailwindcss.com/GTaRErVisj?file=css

![image](https://user-images.githubusercontent.com/485747/108459257-6944aa00-72ca-11eb-8bb9-3c3e552184ab.png)

Hope it helps 👍
